tools/cockpit.spec

```diff
@@ -374,10 +374,10 @@ The Cockpit component for managing storage.  This package uses Storaged.
 
 %package ostree
 Summary: Cockpit user interface for rpm-ostree
-%if 0%{?rhel}
-Requires: rpm-ostree-client >= 2015.11-1
-%else
+%if 0%{?fedora} > 0 && 0%{?fedora} < 24
```

Comment:
What about putting `Requires: /usr/libexec/rpm-ostreed` into a %centos guard so that we know which OS it is for, and can use better ways for the other OSes?